static/js/brand-store/Settings/Settings.test.js

```diff
@@ -15,59 +15,105 @@ jest.mock("react-router-dom", () => ({
   useParams: jest.fn().mockReturnValue({ id: "test" }),
 }));
 
-describe("<Settings />", () => {
-  const testState = {
+let initialState = {};
+
+const setupMockSelector = (state) => {
+  useSelector.mockImplementation((callback) => {
+    return callback(state);
+  });
+};
+
+beforeEach(() => {
+  initialState = {
     currentStore: {
       currentStore: {
-        id: "test",
+        id: "test-id",
         private: true,
         "manual-review-policy": "allow",
       },
     },
   };
+});
 
-  beforeEach(() => {
-    useSelector.mockImplementation((callback) => {
-      return callback(testState);
-    });
-  });
+test("the 'is public' checkbox should not be checked when the current store is set to private", () => {
+  setupMockSelector(initialState);
 
-  it("should render the Settings view", () => {
-    render(
-      <Provider store={store}>
-        <Settings />
-      </Provider>
-    );
-    expect(document.querySelector("#is_public")).toBeInTheDocument();
-  });
+  render(
+    <Provider store={store}>
+      <Settings />
+    </Provider>
+  );
 
-  test("the 'is public' checkbox should not be checked when the current store is set to private", () => {
-    render(
-      <Provider store={store}>
-        <Settings />
-      </Provider>
-    );
-    expect(document.querySelector("#is_public").checked).toEqual(false);
-  });
+  expect(document.querySelector("#is_public").checked).toEqual(false);
+});
 
-  test("the 'is public' checkbox should be checked when the current store is not set to private", () => {
-    useSelector.mockImplementation((callback) => {
-      return callback({
-        currentStore: {
-          currentStore: {
-            id: "test",
-            private: false,
-            "manual-review-policy": "allow",
-          },
-        },
-      });
-    });
-
-    render(
-      <Provider store={store}>
-        <Settings />
-      </Provider>
-    );
-    expect(document.querySelector("#is_public").checked).toEqual(true);
-  });
+test("the 'is public' checkbox should be checked when the current store is not set to private", () => {
+  initialState.currentStore.currentStore.private = false;
+  setupMockSelector(initialState);
+
+  render(
+    <Provider store={store}>
+      <Settings />
+    </Provider>
+  );
+
+  expect(document.querySelector("#is_public").checked).toEqual(true);
```

Comment:
You can use `toBeChecked` here https://github.com/testing-library/jest-dom#tobechecked

Reply:
Done